UI/css/ledgersmb.css

```diff
@@ -73,11 +73,15 @@ table.warning {
     border: 0;
 }
 
-#company_info, #version_info, #login_info {
+#version_info {
    font-weight: bold;
    font-size: 9pt;
 }
 
+#company_info, #login_info {
```

Comment:
If these elements are also styled in 'expanded.css', shouldn't this be moved to there as well?